src/Adapter/Guzzle/GuzzleAdapter.php

```diff
@@ -31,7 +31,7 @@ public function __construct(
             ['headers' => [
                 'Fastly-Key' => $fastlyKey,
                 'Accept'     => 'application/json',
-                'User-Agent' => 'fastly-php'
+                'User-Agent' => 'fastly-php-v' . Fastly::VERSION
```

Comment:
The intention here is to interpolate the version to the end of a the preceding string.